### DISCUSSION TITLE
Build version for Structurizr onpremises

### DISCUSSION THREAD

**Discussion by NathanDotTo** [structurizr/onpremises#50] [Q&A]
### Question

Hi, I am creating a Maven pom.xml for my Structurizr onpremises build. I want to add the correct version number for the artefacts, but I can’t see a version number in the Gradle configs. That implies, I think, that the version is set when the Gradle build is executed, but I can’t see where that is either. 

Can anyone help here please?

Many thanks

Nathan

**Comment by simonbrowndotje** [comment]
> I think, that the version is set when the Gradle build is executed

Correct ... the cloud service, on-premises, and Lite build numbers are all synced via a private build pipeline that updates the [build.properties file](https://github.com/structurizr/onpremises/blob/main/src/main/resources/build.properties), although this isn't committed to git.
